compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -371,6 +371,12 @@ error.missing.required.record.field=\
 error.default.values.not.allowed.for.optional.fields=\
   a default value specified for optional field ''{0}''
 
+error.never.type.not.allowed.for.required.fields=\
+  field ''{0}'' is defined as a never-typed required field
+
+error.never.type.not.allowed.as.variable.type=\
+  variable ''{0}'' is not allowed to have type ''never''
```

Comment:
How about,
```
''never''-typed variable definitions not allowed
```
or
```
defining variables of type ''never'' not allowed
```
or 
```
variables of type ''never'' cannot be defined
```

Reply:
Seems like
`''never''-typed variable definitions not allowed`
fits better